File: android/src/main/java/com/github/musicode/RNTChatViewModule.java

package com.github.musicode;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

public class RNTChatViewModule extends ReactContextBaseJavaModule {

  private final ReactApplicationContext reactContext;

  public RNTChatViewModule(ReactApplicationContext reactContext) {
    super(reactContext);
    this.reactContext = reactContext;
  }

  @Override
  public String getName() {
    return "RNTChatView";
  }

}

File: android/src/main/java/com/github/musicode/chatview/RNTChatViewModule.kt
package com.github.musicode.chatview

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule

class RNTChatViewModule(private val reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String {
        return "RNTChatView"
    }

}